mypy/semanal.py

```diff
@@ -3546,6 +3551,14 @@ def visit_file(self, file: MypyFile, fnam: str, mod_id: str, options: Options) -
                 else:
                     typ = UnionType([UnboundType('__builtins__.str'),
                                      UnboundType('__builtins__.unicode')])
+            elif name == '__path__':
+                # __path__ is only defined on __init__.py files
+                if os.path.basename(fnam).lower() not in ('__init__.py', '__init__.pyi'):
+                    continue
+                # typ = self.sem.named_type('builtins.list', [self.sem.named_type('builtins.str')])
+                # typ = UnboundType('__builtins__.list', args=[UnboundType('__builtins__.str')])
+                # self.visit_import(Import(ids=[('typing', None)]))
+                typ = UnboundType('typing.List', args=[UnboundType('__builtins__.str')])
```

Comment:
Already tried that; it doesn't work. See the discussion on the first version of the commit. The error in that case is: 'error: "list" is not subscriptable, use "typing.List" instead`.